libs/blocks/chart/chart.js

```diff
@@ -147,32 +164,40 @@ const handleIntersect = (chart, chartOptions) => (entries, observer) => {
   });
 };
 
+const getColors = (authoredColor) => {
+  const colorList = Object.values(colorPalette);
+  if (isNullish(authoredColor) || !colorPalette.hasOwnProperty(authoredColor)) return colorList;
+  const colorIndex = colorList.indexOf(colorPalette[authoredColor]);
+  return colorList.concat(colorList.splice(0, colorIndex));
```

Comment:
This is my preferred formatting. Variables should be in a block together but have a newline after the block. Similarly, conditionals should have a newline after. I believe this is what the airbnb styleguide suggests
```suggestion
  const colorList = Object.values(colorPalette);

  if (isNullish(authoredColor) || !colorPalette.hasOwnProperty(authoredColor)) return colorList;

  const colorIndex = colorList.indexOf(colorPalette[authoredColor]);

  return colorList.concat(colorList.splice(0, colorIndex));
```

Reply:
Thanks for the feedback. I think I got confused because I was seeing varying formatting in other parts of the code. I'll go ahead and make this change and fix the formatting in some of the other functions I haven't modified.